src/unix/bsd/netbsdlike/netbsd/mod.rs

```diff
@@ -2862,6 +2862,22 @@ extern "C" {
     pub fn getrandom(buf: *mut ::c_void, buflen: ::size_t, flags: ::c_uint) -> ::ssize_t;
 
     pub fn reboot(mode: ::c_int, bootstr: *mut ::c_char) -> ::c_int;
+
+    #[link_name = "___lwp_park60"]
```

Comment:
Do you have a link/source referencing the link name? I can't seem to find it mentioned anywhere

Reply:
[here for example](https://github.com/NetBSD/src/blob/889f3bb010ad20d396fb291b89f202288dac2c87/include/lwp.h#L57)

Reply:
note that rust uses directly [the "real" symbol](https://github.com/rust-lang/rust/blob/e9e27ab0cf3759e420eeaa6bab7c7d454d9c10cd/library/std/src/sys/pal/unix/thread_parking.rs#L33).

Reply:
Perfect, thank you.